tools/wpt/expectation.json

```diff
@@ -4237,7 +4227,6 @@
     "opening-handshake": {
       "003-sets-origin.worker.html": false
     },
-    "Create-on-worker-shutdown.any.worker.html": false,
```

Comment:
This test hangs, since `t.done()` is never called. On main it panics and satisfies the expected file failure but that panic fixed in this PR. Once #14318 is addressed we can revert to this since it would hit the `the event loop run out of tasks during the test` thingy.